### PR TITLE
Update Hero.astro

### PR DESCRIPTION
**Pull request by krishguptadev**
#123 

<a href="https://gitpod.io/#https://github.com/web3community/devprotocol.xyz/pull/125"></a>

### CODE REVIEW

File: src/components/Creators/Hero.astro [hunk 1]
```diff
@@ -11,7 +11,7 @@
     lg:flex-row-reverse
   "
 >
-  <div class="w-auto items-center mx-12">
+  <div class="w-auto items-center m-0 lg:mx-12">
```
Comment (krishguptadev) [inline src/components/Creators/Hero.astro:14]:
```suggestion
  <div class="w-auto items-center m-0 lg:mx-20">
```

**Comment by vinzvinci** [comment]
Hmm looks hard to align.

If it is possible, make the 2 pages like the [Patrons](https://deploy-preview-125--devprotocol.netlify.app/patrons/) page: 
image

and same with [creators](https://deploy-preview-125--devprotocol.netlify.app/creators/) image center in mobile
image

**Comment by krishguptadev** [comment]
I think creators is perfect aligned now.

image

**Comment by vinzvinci** [comment]
Yes in mobile @krishguptadev

this is how it looks in desktop 
image

**Comment by vinzvinci** [comment]
> Hmm looks hard to align.
> 
> If it is possible, make the 2 pages like the [Patrons](https://deploy-preview-125--devprotocol.netlify.app/patrons/) page: image
> 
> and same with [creators](https://deploy-preview-125--devprotocol.netlify.app/creators/) image center in mobile image

So I think it is best to copy the Patrons alignment except it's mobile view

**Review by vinzvinci — APPROVED** [review]
Looks good for this issue #124.

Now the current issue is the navbar of Creators and Patrons.

The navbar of Developers page is good

**Comment by krishguptadev** [comment]
> Looks good for this issue #124.
> 
> Now the current issue is the navbar of Creators and Patrons.
> 
> The navbar of Developers page is good

as stated the navbar is evenly aligned based on vw. Can't do anything to it.

**Comment by vinzvinci** [comment]
Hmm this still needs to be fix, we can't let it stay like that. I'll look into it later. U can merge this PR

**Comment by krishguptadev** [comment]
> Hmm this still needs to be fix, we can't let it stay like that. I'll look into it later. U can merge this PR

we can change the nav code to fix this...

**Review by shuklaritvik06 — APPROVED** [review]
LGTM 🎉

**Pull request merged**